set-start.js

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env node
+
+var path = require('path')
+var fs = require('fs')
+
+// use process.env to access npm config environment variables
+var saveRequested = process.env.npm_config_save
+// https://docs.npmjs.com/misc/config#environment-variables
+// https://docs.npmjs.com/misc/config#save
```

Comment:
can you move these two comments above `var saveRequested = process.env.npm_config_save` please?